frontend/src/components/student/GameCardPage/GameCardStyles.js

```diff
@@ -1,27 +1,25 @@
-import { Card, Row } from 'react-bootstrap'
 import styled from 'styled-components'
-import { Content } from '../../App/AppGeneralStyles'
+import { Card } from 'react-bootstrap'
 
-export const GameCardContent = styled(Content)`
-  display: flex;
-  justify-content: center;
-  align-content: center;
-  height: 100%;
-  max-width: 100%;
-`
-
-export const GameCardOptionPick = styled(Card)`
-  height: 100%;
-  background-color: var(--dark-blue);
+export const CustomCard = styled(Card)`
   color: var(--font-color);
-`
+  height: 97%;
+
```

Comment:
Tutaj zgubiło się:
```suggestion
border-radius: 10px;
```

Bez tego jest dodatkowy border poza samą kartę:
![firefox_Rtw3i1B2NI](https://user-images.githubusercontent.com/56168780/184689003-9fa2e8db-2b0d-4bd0-bd6a-590df92bb6a7.png)
![firefox_B3f7eK0hKl](https://user-images.githubusercontent.com/56168780/184689017-7baa349e-7110-4c46-bd20-f79aa2b542f2.png)